calvin/runtime/south/plugins/transports/lib/twisted/twisted_transport.py

```diff
@@ -269,7 +269,23 @@ def _client_connected(self, uri, protocol):
             Callback when the client connects still needs a join to be finnshed
             before we can callback upper layers
         """
+        _log.info("Client connected")
         import socket
+        
+        if uri in self._peers:
+            _log.info("Peer %s already connected" % uri)
+            # Disconnect client localy and remove callbacks
+            
+            class ErrorMessage:
+                def __init__(self, str):
+                    self._str = str
+
+                def getErrorMessage(self):
+                    return self._str
+            #self._peers[uri]._transport._disconnected(ErrorMessage("Connection was closed cleanly."))
```

Comment:
Yes it is, the getErrorMessage function is used at row 208 and 214 when checking the reason of a disconnect.